lib/tmuxinator/project.rb

```diff
@@ -328,9 +328,15 @@ def show_tmux_options
         "show-window-option -g pane-base-index\\;"
     end
 
+    def tmux_new_session_path
+      if root? && Tmuxinator::Config.version >= 1.9
+        "#{Tmuxinator::Config.default_path_option} #{root}"
+      end
+    end
+
     def tmux_new_session_command
       window = windows.first.tmux_window_name_option
-      "#{tmux} new-session -d -s #{name} #{window}"
+      "#{tmux} new-session #{tmux_new_session_path} -d -s #{name} #{window}"
```

Comment:
If I understand correctly, this whole section doesn't really matter because of this
https://github.com/tmuxinator/tmuxinator/blob/0f7e9ffa93c508c45bb0bfd7537a7b5b675e0048/lib/tmuxinator/assets/template.erb#L9

Reply:
It's true that the `-c` option's default is the current working directory.

It doesn't hurt to both `cd` and set `-c` here, even when using `mux start --no-attach [PROJECT]`. I tried this, and the CWD was not changed, despite the `cd` command being in the shell script output with `mux debug --no-attach [PROJECT]`.

Still, I guess we can completely remove `tmux_new_session_path` 🤔

Reply:
I tried removing the `-c` entirely to rely on the current working directory, and the session path still seems to get set properly. I'll update the PR.

Reply:
Should we consider dropping support for tmux < 1.9, and then getting rid of the `cd` instead?

Reply:
I'm not sure what removing the `cd` and relying on `-c` gets us. Keeping it allows us to continue supporting the older tmux versions, which is a plus IMO.

Reply:
fair